modules/fluid_properties/doc/content/source/actions/AddFluidPropertiesDeprecatedAction.md

```diff
@@ -0,0 +1,5 @@
+# AddFluidPropertiesDeprecatedAction
```

Comment:
Do we not have action deprecation macros like we do for moose objects?

Reply:
I don't know - where might those be defined (or where are the moose object ones defined)?

Reply:
in base/Registry.h

Reply:
sadly it looks like we don't currently